packages/turf-directional-mean/index.ts

```diff
@@ -176,7 +176,7 @@ function getLengthOfLineString(line: Feature<LineString>, isPlanar: boolean) {
         return previousValue! + euclideanDistance(coords);
       },
       0
-    );
+    ) as number;
```

Comment:
Similar to my comment on turf-length, if this comes back as undefined (passed an empty feature collection or null geometries) then consider the distance to return should just be zero.

Reply:
If you pass a default value that is a number it will definitely return a number.